core-data/model-resolvers.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { camelCase, upperFirst } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import modelsConfig from './models';
+import { getModelRecord } from './resolvers';
+
+export default modelsConfig.reduce( ( memo, { kind, name } ) => {
```

Comment:
Minor: Just noting that this is a runtime evaluation which could theoretically be pre-evaluated (the sort of thing [prepack](https://prepack.io/) would love to optimize for).